ansible/vars/dev-vagrant.user.yml

```diff
@@ -6,6 +6,7 @@
 
 
 # Require confirmation of the email address of newly registered users.
+# Require site_algolia_app_id and site_algolia_api_key of algolia application.
 # Requires a working mailer setup. Add SMTP credentials in
 # secrets/dev-vagrant.secrets.yml before enabling this setting.
-#site_user_email_confirmation_enabled: true
+# site_user_email_confirmation_enabled: true
```

Comment:
This comment is using intentionally no space after the "#" sign to show that this is not a comment, but a commented-out config setting. This is commonly done in example or preconfigured config files.